### DISCUSSION TITLE
Conflicting messages about double-dash --

### DISCUSSION THREAD

**Discussion by igorpupkinable** [percy/cli#732] [Support] [answered]
Yarn and Percy show conflicting messages about double-dash CLI arguments separator.
```
warning From Yarn 1.0 onwards, scripts don't require "--" for options to be forwarded. In a future version, any explicit "--" will be forwarded as-is to the scripts.
$ /app/node_modules/.bin/percy exec yarn run exec
[percy] Warning: Missing command separator (--), some command options may not work as expected
```

## Environment

- Node version: 16 (lts-buster Docker image)
- `@percy/cli` version: 1.0.0-beta.73
- OS version: Debian 10
- Type of shell command-line [interface]: 
Docker with dashes: `ENTRYPOINT ["yarn", "percy", "exec", "--", "yarn", "run", "exec"]`
or Docker without dashes: `ENTRYPOINT ["yarn", "percy", "exec", "yarn", "run", "exec"]`
NodeJS script exec: `jest --runInBand --config=jest.build.js`

## Details

Conflicting warning messages from Yarn and Percy regarding use of double-dash.

## Debug logs
```
yarn run v1.22.15
warning From Yarn 1.0 onwards, scripts don't require "--" for options to be forwarded. In a future version, any explicit "--" will be forwarded as-is to the scripts.
$ /app/node_modules/.bin/percy exec yarn run exec
[percy] Warning: Missing command separator (--), some command options may not work as expected
[percy] Percy is disabled
[percy] Running "yarn run exec"
$ jest --runInBand --config=jest.build.js
```

**Comment by Robdel12** [comment, marked as answer]
Hey @igorpupkinable! The warning is correct, Yarn is stripping the `--` forwarding arg from the Percy command, which the Percy command needs. You can see that with what's printed: `/app/node_modules/.bin/percy exec yarn run exec` (no `--`). So, you'll want to do a double `-- --`: `percy exec -- -- yarn run exec`.
